### PR TITLE
data:harsitagarwalla187

### PR DESCRIPTION
**Pull request by harsitagarwalla187**

### CODE REVIEW

File: data/harsitagarwalla187.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/HarsitAgarwall5",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/harsit-agarwalla-b2b0871b8",
+      "icon": "FaTwitter"
```
Comment (Panquesito7) [inline data/harsitagarwalla187.json:16]:
```suggestion
      "icon": "FaLinkedIn"
```

File: data/harsitagarwalla187.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/harsitagarwalla187",
+      "icon": "FaTwitter"
```
Comment (Panquesito7) [inline data/harsitagarwalla187.json:21]:
```suggestion
      "icon": "FaGitHub"
```

**Review by Panquesito7 — COMMENTED** [review]
Not sure if you want to update the icons.

File: data/harsitagarwalla187.json (hunk 2 above)
Comment (harsitagarwalla187) [reply data/harsitagarwalla187.json:21]: Thanks. I forgot to made that changes.

**Comment by harsitagarwalla187** [comment]
Thanks @Panquesito7 . I have resolved it.

File: data/harsitagarwalla187.json (hunk 1 above)
Comment (eddiejaoude) [reply data/harsitagarwalla187.json]: Good spot 👍

**Comment by eddiejaoude** [comment]
Great collaboration 👍

**Pull request merged**